nansat/tests/test_vrt.py

```diff
@@ -545,5 +557,44 @@ def test_get_super_vrt_and_copy(self):
         self.assertFalse(data is None)
         self.assertTrue(np.all(data == array))
 
+    def test_filename_warning(self):
+        vrt = VRT()
+        with warnings.catch_warnings(record=True) as w:
+            vrt_filename = vrt.fileName
+            self.assertEqual(vrt_filename, vrt.filename)
+
+    def test_get_sub_vrt0(self):
+        vrt1 = VRT()
+        vrt2 = vrt1.get_sub_vrt()
+        self.assertEqual(vrt1, vrt2)
+
+    def test_get_sub_vrt3(self):
```

Comment:
When I tried to understand what is going on I went to docstring of get_super_vrt method and found: `copy of self in vrt.vrt and change references from vrt to vrt.vrt`. Seems like this is very important method, so could you please include a better documentation of the <get_super_vrt> method to this PR

Reply:
Fixed in 96fbd90